.github/workflows/zz_generated.create_release.yaml

```diff
@@ -118,8 +118,15 @@ jobs:
           echo "::add-path::/opt/bin"
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Ensure non-reference version
+        id: non_ref-version
+        run: |
+          if [[ "${{ needs.gather_facts.outputs.version }}" =~ ^[0-9]+.[0-9]+.[0-9]+-[0-9]+$ ]]; then
```

Comment:
How about adding a new output like `update_project_go: true/false` in `gather_facts` and skipping the whole job (`update_project_go`) if it's set to false?

Reply:
It is used also in the validation before tagging so I keep it as `refversion`